Azure/sonic-buildimage
# [submodule] Update sonic-swss submodule

**- Why I did it**
To include latest VNET changes in upstream code.

The following changes are added as part of this submodule update:
- 62e6c5a [bitmap_vnet] Remove BMTOR implementation (#1494)
- 582beae [FPMSYNCD] EVPN Type5 prefix handling in FPMSYNCD (#1326)
- d9ca6e2 [vstest] Resolve intermittent timing failure in test_fgnhg.py (#1495)
- a89b8ed [sub intf] Inherit parent port mtu changes (#1479)

**- How I did it**
Updated ```sonic-swss``` submodule pointer

**- How to verify it**
Build an image and run tests to make sure all is up and running.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

[submodule] Update sonic-swss submodule
